react/Header/UserAccount/UserAccount.less

```diff
@@ -62,7 +63,7 @@
   .smallDeviceOnly({
     transform: translateY(@grid-row-height * -1);
     transition: opacity 0.25s ease, transform 0.25s ease;
-    top: (@grid-row-height * 7); // header height
+    top: @headerHeightMobile;
```

Comment:
New version of styling complains on space after declaration, so instead of having a comment a decided to have a variable here, as it's anyway more correct. Scope creep